.haml-lint_todo.yml

```diff
@@ -1,16 +1,12 @@
 # This configuration was generated by
 # `haml-lint --auto-gen-config`
-# on 2023-07-11 23:58:05 +0200 using Haml-Lint version 0.48.0.
+# on 2023-07-12 14:52:52 -0400 using Haml-Lint version 0.48.0.
```

Comment:
Aside: I did file something upstream to see if they'd adopt rubocops quieter regeneration without the stamps and counts